src/EditorFeatures/Core.Wpf/Completion/Presentation/CustomCommitCompletion.cs

```diff
@@ -25,6 +25,8 @@ internal sealed class CustomCommitCompletion : Completion3, ICustomCommit
         public CustomCommitCompletion(
             CompletionPresenterSession completionPresenterSession,
             CompletionItem completionItem)
+            : base(displayText: null, insertionText: null, description: null,
+                   iconMoniker: default, suffix: completionItem.InlineDescription)
```

Comment:
It's interesting that property `Suffix` isn't virtual, is this intentional?

Reply:
My guess is 'no'.  but it's not our api, so i'm just plugging in with teh existing publicly supported api surface area.